autoload/lsp/lsp.vim

```diff
@@ -111,6 +111,17 @@ export def ShowServers()
   endfor
 enddef
 
+# Get LSP server status for filetype 'ftype'
+# Return true if running, or false if not found or not running
+export def ServerStatus(ftype: string): bool
```

Comment:
As this function returns the server running status, can you change the name of this function to ServerRunning?